src/pybel/io/nodelink.py

```diff
@@ -29,9 +30,13 @@
 ]
 
 
-def to_nodelink(graph: BELGraph) -> Mapping[str, Any]:
```

Comment:
While this new kind of export might follow the schema of to_nodelink, it no longer conveys the information that a nodelink graph should. Please break it out into its own submodule in `pybel.io`. Therefore this needs a new name, a significant explanation of why someone would want to represent edges this way (since I still disagree with it), and test case(s).